src/main/java/com/enderio/core/common/BlockEnder.java

```diff
@@ -167,6 +168,20 @@ protected boolean shouldDoWorkThisTick(World world, int x, int y, int z, int int
     }
   }
 
+  @Override
+  public boolean hasComparatorInputOverride() {
+    return IComparatorOutput.class.isAssignableFrom(teClass);
```

Comment:
I would still prefer it be cached to maintain consistency with how the IProgressTile checks work